plasmapy/atomic/tests/test_atomic.py

```diff
@@ -619,7 +624,9 @@ def test_half_life_u_220():
             atomic_TypeError_bad_arguments))
 def test_atomic_TypeErrors(function, argument):
     with pytest.raises(TypeError):
-        function(argument)
+        with warnings.catch_warnings():
+            warnings.simplefilter('ignore')
```

Comment:
Hmm. What's the goal here? Perhaps this could be simply replaced with two layers of `pytest.raises` and `pytest.warns`?

Reply:
This is a remnant from the rabbit hole I went down 🐰 🕳 , so I removed it.  (I actually went down a rabbit hole in real life once, so that might be where I got some of my coding habits, perhaps?)